components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -110,6 +163,13 @@ service NodesService {
 			action: "infra:nodes:rerun"
 		};
 	};
+
+	/*
+	Bulk delete
+
+	Deletes a set of nodes given a filter query to match. 
```

Comment:
```suggestion
	Deletes a set of nodes that match a filter. 
```
Would this delete all of the nodes that match a filter, or can you limit the range? Is it possible to use more than one filter to delete a specific subset?

Reply:
yes, it deletes all nodes that match a filter. and yes, you can use more than one filter. it uses the same query object that list does so that list can be used as a "dry run" of what nodes will be deleted